pkg/cni/cilium/cilium.go

```diff
@@ -194,6 +194,11 @@ func GetAppInstallOverrideValues(cluster *kubermaticv1.Cluster, overwriteRegistr
 			// we run Cilium as non-exclusive CNI to allow for Multus use-cases
 			"exclusive": false,
 		},
+		"podSecurityContext": map[string]any{
+			"seccompProfile": map[string]any{
+				"type": "RuntimeDefault",
```

Comment:
kkp tests expect pod seccomp profile to be set by default

[upstream chart allows that] (https://github.com/cilium/cilium/blob/a67489466c715854c4c472c939d32d35df3fe748/install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml#L80C24-L80C42)

https://github.com/kubermatic/kubermatic/blob/31f9faf5c8185733e59e2485dbd38180d4e2c51d/cmd/conformance-tester/pkg/tests/usercluster_controller.go#L149

Reply:
and we actually need it on all
https://github.com/cilium/cilium/commit/5aa1c236915444771e14ffb5fbe532e4e5de1c2d
```
{  function did not succeed after 3 attempts: expected seccomp profile on Pod kube-system/cilium-operator-85b8f66cd-hgkhq, got none
expected seccomp profile on Pod kube-system/hubble-generate-certs-g5pl8, got none
expected seccomp profile on Pod kube-system/hubble-relay-5ccf97b646-f6gnt, got none
expected seccomp profile on Pod kube-system/hubble-ui-869b75b895-trq2t, got none}
```

Reply:
configured in properly in https://github.com/kubermatic/kubermatic/pull/12752/commits/15cd1ef62ea4abbb849ea8693485fc65a0a90ff5